platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -113,7 +113,48 @@
         },
         loadEntity;
 
-    controllers.controller('MdsBasicCtrl', function ($scope, $location, $route, Entities, MDSUtils) {
+    controllers.controller('MdsEmbeddedCtrl', function ($scope, MDSUtils) {
```

Comment:
Is this controller still used in MDS? If yes leave the name, eventually rename it to MdsEmbeddableCtrl